platform/ios/src/NSOrthography+MGLAdditions.m

```diff
@@ -3,6 +3,10 @@
 @implementation NSOrthography (MGLAdditions)
 
 + (NSString *)mgl_dominantScriptForMapboxStreetsLanguage:(NSString *)language {
```

Comment:
By the way, my earlier feedback was indirectly suggesting that `NS_ASSUME_NONNULL_*` macros would be added to NSString+MGLAdditions.h and the parameter and return type of this method would be marked `nullable`.

Reply:
Updated